conf/conf.yaml

```diff
@@ -0,0 +1,9 @@
+openmetadata_endpoint: "http://localhost:8585/api"
```

Comment:
Hi.
I like it better when the full endpoint is given and not constructed:
- This would prevent problems if the endpoint changes from "http://hostname:port/api" to (for example) "https://hostname:port/api/v2"
- I think it is the more common approach. See, for example: https://github.com/open-metadata/openmetadata-helm-charts/blob/main/charts/openmetadata/values.yaml#L59

I think that it is better to have all vault configuration parameters under `vault`. The connector is supposed to work without vault. So one can simply omit the entire `vault` section of the configuration.